imap_processing/cdf/config/imap_codice_l1a_variable_attrs.yaml

```diff
@@ -13,6 +14,35 @@ codice_support_attrs: &support_default
     <<: *default
     VAR_TYPE: support_data
 
+# <=== Coordinates ===>
+energy_attrs:
+    <<: *default
+    CATDESC: Energy per charge (E/q) sweeping step
+    FIELDNAM: Energy step
+    FORMAT: I3
+    LABLAXIS: energy
+    UNITS: ' '
```

Comment:
MeV / nuc?

Reply:
I think this is actually unitless because it is an energy _step_ index (i.e. 1, 2, 3, ... 127).

Reply:
Ahh, OK. Maybe LABLAXIS should be "step #" / "energy step" or something like that then instead of just plain energy?